docs/_includes/documentation-menu.html

```diff
@@ -16,6 +16,7 @@
 			<li><a href="{{docu}}/features/rest.html">REST API</a></li>
 			<li><a href="{{docu}}/features/dsl.html">Textual Configuration</a></li>
 			<li><a href="{{docu}}/features/internationalization.html">Internationalization</a></li>
+			<li><a href="{{docu}}/features/providedServices.html">Provided Services</a></li>
```

Comment:
"Provided Services" sounds weird.
How about something like "Utilities", "System Services" or "Framework Services/Utilities"?
Note that not everything here might be real "services" (e.g. our caching classes, etc.).
@maggu2810 Any good suggestion or preference from your end?